lib/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -33,81 +45,55 @@ export const DateTimePickerToolbar: React.FC<ToolbarComponentProps> = ({
   hideTabs,
   dateRangeIcon,
   timeIcon,
-  onChange,
+  title = 'SELECT DATE & TIME',
```

Comment:
```suggestion
  title = 'Select date & time',
```

I'm wondering if we should rename the prop to `titleText` to something that starts to look like a convention:
- backIconButtonText
- labelRowsPerPage
- labelDisplayedRows
- nextIconButtonText
- getLabelText
- clearText
- closeText
- loadingText
- noOptionsText
- openText

Reply:
We also need to be careful when using DOM attribute names.

Reply:
I thought this stuff should be filled from `label` coming to input. And then be overridable with custom prop